From 50bb56193f97b7dd683af710e4227a682e58df2c Mon Sep 17 00:00:00 2001
From: Mario Schuknecht <mario.schuknecht@dresearch-fe.de>
Date: Tue, 14 Jan 2020 09:11:44 +0100
Subject: [PATCH] meson dependency HYP-22586

In yocto warrior branch there is not meson version 0.50 but meson
version 0.49. With meson version 0.49 libsoup 2.68.3 can be built
successfully.

Signed-off-by: Mario Schuknecht <mario.schuknecht@dresearch-fe.de>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index aba8cec..2fcfd0f 100644
--- a/meson.build
+++ b/meson.build
@@ -1,6 +1,6 @@
 project('libsoup', 'c',
         version: '2.68.3',
-        meson_version : '>=0.50',
+        meson_version : '>=0.49',
         license : 'LGPL2',
         default_options : 'c_std=c89')
 
-- 
2.16.4

